fissix/tests/test_fixers.py

```diff
@@ -7,7 +7,7 @@
 
 # Local imports
 from fissix import pygram, fixer_util
-from fissix.tests import support
+from . import support
```

Comment:
This feels weird going from absolute to relative paths when all the other changes are from relative to absolute. Is there a reason this needs to be different?

Reply:
It's separating the two modules, which means two kinds of changes to imports:
1. Imports from the tests to `fissix` need to be explicit, because the tests may not be a child of the `fissix` module.
2. Imports within the tests need to be relative, because they may not be called `fissix.tests`.